blog-service/2024-02-29-apps.md

```diff
@@ -0,0 +1,20 @@
+---
+title: HipChat Webhook Connection Deprecated (Apps)
+image: https://help.sumologic.com/img/sumo-square.png
+keywords:
+  - webhooks
+  - connections
+  - webhook connections
+hide_table_of_contents: true
+authors:
+  - url: https://help.sumologic.com/release-notes-service/rss.xml
+    image_url: /img/release-notes/rss-orange.png
+---
+
+Our webhook connection for HipChat has been deprecated and is now obsolete.
+
+For the full list of our supported webhook connections, see [Sumo Logic Webhook Connections](/docs/alerts/webhook-connections).
+
+:::note
+Our built-in HipChat CSE Action was deprecated in August 2022.
```

Comment:
It might be a good idea to link text "was deprecated in August 2022" to the release note where it was announced:
https://help.sumologic.com/release-notes-cse/2022/12/31/#july-8-2022---application-update

I just noticed that the July 8 2022 release note is duplicated. You should remove the second instance.

Reply:
AH, good catch! will fix